include/sys/vdev_raidz_impl.h

```diff
@@ -141,6 +141,12 @@ extern const raidz_impl_ops_t vdev_raidz_ssse3_impl;
 #if defined(__x86_64) && defined(HAVE_AVX2)	/* only x86_64 for now */
 extern const raidz_impl_ops_t vdev_raidz_avx2_impl;
 #endif
+#if defined(__x86_64) && defined(HAVE_AVX512F)	/* only x86_64 for now */
```

Comment:
Yes, 8 available regs makes it hard to implement reconstruction (hard but not impossible). However, with ABD change reg pressure is somewhat lower, so if there's demand, 32bit can be revisited.